examples/02_meta-analyses/plot_simulated_data.py

```diff
@@ -0,0 +1,228 @@
+# foci than another study.
+# You can also:
+#   - manually specify the sample sizes for each study (i.e., ``sample_size``).
+#   - set the number of false positive foci for each study (i.e., ``n_noise_foci``).
+#   - set the weighed probability of each foci being chosen (i.e., ``foci_weights``).
+#   - set the dispersion of the gaussian spread for each foci (i.e., ``fwhm``).
+#   - set the specific ground truth foci (i.e., ``foci_coords``).
+
+num_studies = 30
+n_foci = [4] * num_studies
+n_foci[0] = 10
+sample_sizes = [30] * num_studies
+sample_sizes[0] = 300
+n_noise_foci = [10] * num_studies
+n_noise_foci[0] = 25
+foci_weights = [1, 0.1, 0.5, 0.75]
+fwhm = [6.0, 12.0, 10.0, 8.0]
+_, manual_dset = create_coordinate_dataset(
+    n_foci=n_foci,
+    fwhm=fwhm,
+    sample_size=sample_sizes,
+    n_studies=num_studies,
+    foci_coords=ground_truth_foci,
+    n_noise_foci=n_noise_foci,
+    foci_weights=foci_weights,
+)
+
+###############################################################################
+# Analyze and plot manual dataset
+# -------------------------------
+
+analyze_and_plot(manual_dset, ground_truth_foci)
+
+###############################################################################
+# Control percentage of studies with the foci of interest: Strategy #1
+# --------------------------------------------------------------------
+# There are two ways you can create a dataset where
+# a focus is only present in some of the studies.
+# One way is to have two ground truth foci specified
+# with ``foci_coords``, but only one foci specified
+# for ``n_foci``.
+# The relative probability of the two foci appearing
+# can be controlled with ``foci_weight``.
+# For example the weights ``[0.9, 0.1]`` mean
+# the first foci is 90% likely to appear in a study and
+# the second foci has a 10% chance of appearing in a particular
+# study.
+
+_, two_foci_dset = create_coordinate_dataset(
+    n_foci=1,
+    fwhm=10.0,
+    sample_size=30,
+    sample_size_interval=10,
+    n_studies=30,
+    foci_coords=ground_truth_foci[0:2],
+    n_noise_foci=0,
+    foci_weights=[0.9, 0.1],
+)
+
+###############################################################################
+# Analyze and plot two foci dataset
+# ---------------------------------
+
+analyze_and_plot(two_foci_dset, ground_truth_foci[0:2])
+
+###############################################################################
+# Control percentage of studies with the foci of interest: Strategy #2
+# --------------------------------------------------------------------
+# Another method to control what percentage of studies contain
+# a focus is to have one focus specified in ``foci_coords``
+# and have a list of zeros and ones in ``n_foci`` to specify
+# which studies contain the focus of interest.
+# The caveat of this strategy is that each study must have at least
+# one coordinate to report necessitating ``n_noise_foci`` be set for
+# studies without a ground truth foci.
+
+num_studies = 30
```

Comment:
I think this was just missed in the recent set of changes.

Reply:
fixed!